src/EFCore.MySql/MySqlDbFunctionsExtensions.cs

```diff
@@ -450,5 +450,54 @@ public static class MySqlDbFunctionsExtensions
             => (startDate.HasValue && endDate.HasValue)
                 ? (int?)DateDiffMicrosecond(_, startDate.Value, endDate.Value)
                 : null;
+
+        /// <summary>
+        ///     <para>
+        ///         An implementation of the SQL LIKE operation. On relational databases this is usually directly
+        ///         translated to SQL.
+        ///     </para>
+        ///     <para>
+        ///         Note that if this function is translated into SQL, then the semantics of the comparison will
+        ///         depend on the database configuration. In particular, it may be either case-sensitive or
+        ///         case-insensitive. If this function is evaluated on the client, then it will always use
+        ///         a case-insensitive comparison.
+        ///     </para>
+        /// </summary>
+        /// <param name="_">The DbFunctions instance.</param>
+        /// <param name="matchExpression">The property of entity that is to be matched.</param>
+        /// <param name="pattern">The pattern which may involve wildcards %,_,[,],^.</param>
+        /// <returns>true if there is a match.</returns>
+        public static bool Like<T>(
+            [CanBeNull] this DbFunctions _,
```

Comment:
Nitpick: It feels very strange to call this parameter `_` (which implies a "discard"), but then use it immediately on line 474. `functions` might be a better name.

I do see that this has precedence in this file, but I don't think it's a good precedent to follow.

Reply:
Now I no longer use `_` immediately 😄